docs/consent.md

```diff
@@ -59,8 +59,9 @@ cookies as storage you need to set the `store` option to `consent:Cookie`.
 Example: 
 
 	90 => array(
-		'class' 	=> 'consent:Consent', 
-		'store' 	=> 'consent:Cookie', 
+		'class' 	       => 'consent:Consent',
+                'identifyingAttribute' => 'uid',
```

Comment:
Indenting seems a bit off here?

Reply:
Mine was actually fine, but the original document is full of tabs